build/testing/integration.go

```diff
@@ -21,6 +21,12 @@ var (
 	protocolPorts = map[string]string{"http": "8080", "grpc": "9000"}
 	replacer      = strings.NewReplacer(" ", "-", "/", "-")
 	sema          = make(chan struct{}, 6)
+
+	defaultCases = map[string]testCaseFn{
+		"api":           api,
+		"fs/local":      local,
+		"import/export": importExport,
+	}
```

Comment:
I wonder if we should call this variable `allCases` instead. I feel like "default" has a different meaning connotatively.